Modules/_elementtree.c

```diff
@@ -1339,12 +1366,18 @@ _elementtree.Element.findall
     path: object
     namespaces: object = None
 
+Find all matching subelements by tag name or path.
+
+*path* is a string having either an element tag or an XPath,
+*namespaces* is an optional mapping from namespace prefix to full name.
+
+Returns list containing all matching elements in document order.
```

Comment:
Please consistently use the imperative mood for reference prose. Ditto for the rest of the PR.

```suggestion
Return a list containing all matching elements in document order.
```